_redirects

```diff
@@ -44,6 +44,11 @@
 /guides/js-programming/ses/ses-guide.html https://github.com/endojs/endo/blob/HEAD/packages/ses/docs/guide.md 
 /guides/js-programming/ses/ses-reference https://github.com/endojs/endo/blob/HEAD/packages/ses/docs/reference.md 
 /guides/js-programming/ses/ses-reference.html https://github.com/endojs/endo/blob/HEAD/packages/ses/docs/reference.md 
+/guides/orchestration/getting-started/key-concepts /guides/orchestration/key-concepts
```

Comment:
we also need to add corresponding html entries

Reply:
oh yes

Reply:
I wonder if this can automated - at least for changes in navbar/sidebar config files.

Reply:
Fixed. Have a careful look though, I have never added to `_redirects` before.